crates/modelardb_server/src/optimizer/model_simple_aggregates.rs

```diff
@@ -762,9 +776,9 @@ impl Accumulator for ModelSumAccumulator {
         unreachable!()
     }
 
-    /// Return the current state of the [`Accumulator`].
+    /// Return the current state of the [`Accumulator`]. It must match
+    /// [SumAccumulator](repository/datafusion/physical-expr/src/aggregate/sum.rs).
```

Comment:
is it not necessary to have the full link to the GitHub repository?

Reply:
For some reason only `MinAccumulator`, `MaxAccumulator`, and `AvgAccumulator` are public, so as far as I know, I cannot link to `SumAccumulator` through doc comments. I decided not to include the exact line in the link as it will change in the future.